velox/exec/tests/NestedLoopJoinTest.cpp

```diff
@@ -77,19 +64,19 @@ class NestedLoopJoinTest : public HiveConnectorTestBase {
         size, [start](auto row) { return start + row; });
   }
 
+  void runSingleAndMultiDriverTest(
```

Comment:
Please remove this function. We should avoid a function which does multiple things, it is confusing. This function is not accepting num of drivers as param either. Call runTest directly in the tests.